Dockerfile.ci

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.10
+RUN apt-get update && apt-get -y install iptables
+
+RUN go get github.com/tools/godep \
```

Comment:
don't think godep is still used in this repo

Reply:
will remove it and test it once to make sure there is no failure.